inc/validation.php

```diff
@@ -21,6 +21,14 @@ public function rules( RW_Meta_Box $object ) {
 	public function enqueue() {
 		wp_enqueue_script( 'rwmb-validation', RWMB_JS_URL . 'validation.min.js', [ 'jquery', 'rwmb' ], RWMB_VER, true );
 
+		$locale       = determine_locale();
+		$locale_short = substr( $locale, 0, 2 );
+		$locale       = file_exists( RWMB_DIR . "js/validation/i18n/messages_$locale.js" ) ? $locale : $locale_short;
+
+		if ( file_exists( RWMB_DIR . "js/validation/i18n/messages_$locale.js" ) ) {
+			wp_register_script( 'rwmb-validation-i18n', RWMB_JS_URL . "js/validation/i18n/messages_$locale.js", ['rwmb-validation'], RWMB_VER, true );
```

Comment:
Should we enqueue this script instead of just registering it?

Reply:
Yes you are right. I see that this PR has already been merged. Should I create a new one for this fix?